bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -675,9 +677,12 @@ void BCStateTran::saveReservedPage(uint32_t reservedPageId, uint32_t copyLength,
 
 // TODO(TK) check if this function can have its own transaction(bftimpl)
 void BCStateTran::zeroReservedPage(uint32_t reservedPageId) {
-  LOG_DEBUG(logger_, reservedPageId);
+  if (isFetching()) {
+    LOG_WARN(logger_, "Failed to zero reserved page" << KVLOG(reservedPageId));
```

Comment:
nit:
I would write in all LOG_WARN (line 661 as well):
"Saving reserved page is not allowed during State Transfer" << KVLOG(reservedPageId)
"Zeroing ..." << KVLOG(reservedPageId)

Reply:
Thanks. Addressed.